src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.length) {
+    return {};
+  }
+
+  const stringToObject = sourceString
+    .split(';')
+    .map((value) => value.split(':').map((spaces) => spaces.trim()))
+    .reduce((styleObject, key) => {
+      if (key[0] && key[1]) {
+        styleObject[key[0]] = key[1];
+      }
+
+      return styleObject;
```

Comment:
The `reduce` callback function's parameter `styleObject` is being directly modified, which is not recommended. Instead, you should return a new object that includes all previous styles. This can be achieved by using object spread syntax or Object.assign to avoid mutations.